src/glossary.js

```diff
@@ -145,6 +145,7 @@ Glossary.prototype.handleTermTouch = function(e) {
   if (e.which === KEYCODE_ENTER || e.type === 'click') {
     if (selectorMatches(e.target, '[data-term]')) {
       this.show();
+      this.$selectedTerm = $(e.target);
```

Comment:
Sorry, I forgot to say how this library doesn't require jQuery. This works because the page we're testing on already has it, but this should be refactored to not depend on it.